src/docs/extend/innerblocks.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Innerblocks
+description: How to use innerblocks in wp-lemon
+---
+
+
+By using innerblocks, you can nest blocks within other blocks. 
+This allows you to create more complex and flexible layouts by placing blocks inside parent blocks.
+
+> **Note:**
+> A single block can only contain one `InnerBlocks` component.
+
+## Building your template
+When using innerblocks it's common to create a template for the innerblocks. 
+This will prefill the InnerBlocks component when inserted and will make it easier to use for you and your clients. 
+In wp-lemon we do this inside the `block_context`. 
+
+``` php
+
+
+	public function block_context($context): array
+	{
+		$template = [
+			[
+				'core/heading',
+				[
+					'placeholder' => __('Add a title', 'wp-lemon'),
+					'level'       => 3,
+				],
+			],
+			[
+				'core/paragraph',
+				[
+					'placeholder' => __('Add some text', 'wp-lemon'),
+				],
+			],
+		];
+
+		$args = [
+			'InnerBlocks' => self::create_inner_blocks(false, $template),
+		];
+
+		return array_merge($context, $args);
+	}
+
+```
+In the example above we've used a heading and a paragraph block, 
+ofcourse you can add other core blocks or even wp-lemon blocks. You can do this like so:
+
+``` php
+
+	$template = [
+		['acf/content-card'],
+];
+
+```
+
+## Template properties 
+By adding extra parameters in the `create_inner_blocks` function we can add some extra functionality. 
+In the list below you can see each parameter with it's functionality in the right order.
+
+| Parameter     | Function     |
+|--------------|--------------|
+| $allowed_blocks | This parameter allows you to make a selection of blocks that are allowed in the innerblocks |
+| $template | This parameter refers to your innerblocks template  |
+| $classes | This parameter can be used to give custom classnames to your innerblocks container |
+| $orientation | ???? |
+| $templatelock | This parameter gives the ability to lock down the template so no changes can be made |
+
+> **Note:**
+> If a parameter is not given it will default to false.
+
+
+### Allowed blocks
+The `$allowed_blocks` parameter allows you to make a selection of blocks that are allowed in the innerblocks. 
+We used the `card-grid` block from wp-lemon as an example. 
+In this case the we only wanted to allow specific cards we've made. 
+We did this like so:
+ 
+``` php
+	public function block_context($context): array
+	{
+		$template = [
+			['acf/content-card'],
+		];
+
+		$allowed_blocks = apply_filters('wp-lemon/filter/block/card-grid/allowed-blocks', ['acf/content-card', 'acf/image-card', 'acf/price-card']);
```

Comment:
```suggestion
		$allowed_blocks = ['acf/content-card', 'acf/image-card', 'acf/price-card'];
```

Reply:
De filters gebruiken we alleen in wp-lemon zelf zodat je dit vanuit je child kan aanpassen. In het child thema is dat dus niet nodig.